features/support/env.rb

```diff
@@ -89,20 +89,20 @@
 
 # Make these rules throw errors instead of warnings.
 # Rules list: https://github.com/GoogleChrome/accessibility-developer-tools/wiki/Audit-Rules
-Capybara::Accessible::Auditor.severe_rules = %w(
-  'AX_COLOR_01',
-  'AX_FOCUS_01',
-  'AX_FOCUS_02',
-  'AX_FOCUS_03',
-  'AX_VIDEO_01',
-  'AX_ARIA_06',
-  'AX_ARIA_07',
-  'AX_ARIA_11',
-  'AX_ARIA_13',
-  'AX_HTML_01',
-  'AX_TITLE_01',
-  'AX_TEXT_04'
-)
+Capybara::Accessible::Auditor.severe_rules = %w[
```

Comment:
I think we'll probably end up silencing these errors for this PR, so it doesn't get any bigger, then fix them in a subsequent PR.